wagtail/search/backends/database/fallback.py

```diff
@@ -180,7 +180,7 @@ def _do_search(self):
                 **{self._score_field: Value(None, output_field=models.FloatField())}
             )
 
-        return queryset.iterator()
+        return queryset.iterator(2000)
```

Comment:
This is the default value that Django uses if we don't pass anything:

https://docs.djangoproject.com/en/4.1/ref/models/querysets/#iterator

> So long as the QuerySet does not prefetch any related objects, providing no value for chunk_size will result in Django using an implicit default of 2000.

> Using `iterator()` on a queryset that prefetches related objects without providing the `chunk_size` is deprecated. In Django 5.0, an exception will be raise.

Not sure if we want to hardcode it here or put it in a variable (or make it configurable).

Reply:
Happy to hardcode this until a specific request for making it configurable comes up.